src/System.Private.DataContractSerialization/src/System/Runtime/Serialization/ExceptionDataContract.cs

```diff
@@ -872,5 +872,42 @@ private void Read()
                 throw new InvalidOperationException();
             }
         }
+
+        private string GetEscapedElementString(string stringToEscape)
+        {
+            var sb = new StringBuilder();
```

Comment:
Method signature can also be changed to `private void AppendEscapedElementString(string stringToEscape)`.